manifests/admin_bastion/parallel_shells.pp

```diff
@@ -0,0 +1,100 @@
+# @summary Install and configure parallels shells with config synced from the xCAT node.
+#
+# Install and configure parallels shells with config synced from the xCAT node.
+#
+# Automatically included by profile_xcat::admin_bastion
+#
+# @param cron_data
+#   Hash of data to configure the cron to run sync scripts.
+#
+# @param cron_enable
+#   Enable/disable running scripts via cron.
+#
+# @param file_config_genders
+#   Location of genders config file.
+#
+# @param packages_clustershell
+#   Packages required for clustershell (clush).
+#
+# @param packages_genders
+#   Packages required for genders.
+#
+# @param packages_pdsh
+#   Packages required for pdsh.
+#
+# @param script_install_configs
+#   Path to script to install configs that have been synced from the master node.
+#
+# @param script_sync_configs
+#   Path to script to sync configs from master node.
+#
+class profile_xcat::admin_bastion::parallel_shells (
+  Hash          $cron_data,
+  Boolean       $cron_enable,
+  String        $file_config_genders,
+  Array[String] $packages_clustershell,
+  Array[String] $packages_genders,
+  Array[String] $packages_pdsh,
+  String        $script_install_configs,
+  String        $script_sync_configs,
+) {
+  # Install packages
+  Package {
+    ensure => installed,
+  }
+  ensure_packages( $packages_clustershell )
+  ensure_packages( $packages_genders )
+  ensure_packages( $packages_pdsh )
```

Comment:
Should we `include ::epel` somewhere here? Probably not as that doesn't quite fit how we generally do things in cluster environments.

But we should at least call out in the `README.md` that EPEL repos are necessary to install the parallel shells.

Reply:
Added to the README.md.